core/src/main/java/cz/xtf/core/openshift/OpenShifts.java

```diff
@@ -183,22 +203,49 @@ private static String downloadOpenShiftBinary(String version) {
 		File ocTarFile = new File(workdir, "oc.tar.gz");
 		File ocFile = new File(workdir, "oc");
 
+		final String ocUrl = clientLocation + ocFileName;
 		try {
-			URL requestUrl = new URL(clientLocation + "oc.tar.gz");
-			FileUtils.copyURLToFile(requestUrl, ocTarFile, 20_000, 300_000);
+			URL requestUrl = new URL(ocUrl);
+
+			Https.copyHttpsURLToFile(requestUrl, ocTarFile, 20_000, 300_000);
```

Comment:
I don't think we should be using the trust-all https downloader (which is used for tests) to actually download binaries from internet.